themes/digital.gov/src/scss/new/home/_communities.scss

```diff
@@ -43,3 +44,74 @@ $communities-of-practice: (
     @include u-minh(8);
   }
 }
+
+// ==========================================================================
+// Cards view
+// ==========================================================================
+.dg-featured-communities {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+
+  @include at-media("tablet") {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @include at-media("tablet-lg") {
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  .usa-card {
+    max-width: none !important;
+
+    // Make the first card wider for emphasis
+    &:first-child {
+      grid-column: 1 / 3;
+
+      .usa-button {
+        &:hover {
+          background-color: map.get($communities-of-practice, "WCM");
+        }
+      }
+
+      .usa-button--outline {
+        box-shadow: inset 0 0 0 2px map.get($communities-of-practice, "WCM");
+        color: map.get($communities-of-practice, "WCM");
+        &:hover {
+          color: #fff;
+        }
+      }
```

Comment:
Styles are still coupled to `WCM` brand colors and will need to be updated when changing the weight for a featured community card.

Reply:
Just the first card in the list.